src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -139,7 +139,8 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):#pylint:
                       validator=_handle_vm_nics)
 
 register_cli_argument('vm create', 'name', name_arg_type, validator=_resource_not_exists('Microsoft.Compute/virtualMachines'))
-register_cli_argument('vmss create', 'name', name_arg_type, validator=_resource_not_exists('Microsoft.Compute/virtualMachineScaleSets'))
+register_cli_argument('vmss create', 'name', name_arg_type)
+register_cli_argument('vmss create', 'nat_backend_port', default=None, help='Backend port to open with NAT rules.  Defaults to 22 on Linux and 3389 on Windows.')
```

Comment:
this change happened to come for free :), since this property already exists in vm, we set the property if it existed.  Now it exists in both places.

---
In reply to: [76155155](https://github.com/Azure/azure-cli/pull/743#discussion_r76155155) [](ancestors = 76155155)